README.md

```diff
@@ -1,5 +1,8 @@
 # Octokit - GitHub API Client Library for .NET [![Build Status](https://ci.appveyor.com/api/projects/status/github/octokit/octokit.net?branch=master)](https://ci.appveyor.com/project/Haacked15676/octokit-net)
 
+![logo](octokit.png)
```

Comment:
I love it! However, the logo is HUGE and takes up so much space. Let's use a normal image tag and size it to a reasonable size.